plugins/automox/help.md

```diff
@@ -1478,35 +1802,41 @@ Example output:
 
 # Version History
-
-* 2.0.1 - Fix Get Vulnerability Sync Action Set action
-* 2.0.0 - Fix Vulnerability Sync API Actions | `Action`: Added - Delete Vulnerability Sync Action Set | `Action`: Added - Execute Vulnerability Sync Actions | `Action`: Added - List Vulnerability Sync Action Set Issues |`Action`: Added - List Vulnerability Sync Action Set Solutions | `Action`: Added - List Vulnerability Sync Action Sets | `Action`: Added - Get Vulnerability Sync Action Set | `Action`: Updated - Upload Vulnerability Sync File | `Action`: Updated - Get Devices | `Action`: Deleted - Action on Vulnerability Sync Batch | `Action`: Deleted - Vulnerability Sync Task | `Action`: Deleted - Get Vulnerability Sync Batch | `Action`: Deleted - List Vulnerability Sync Batches | `Action`: Deleted - List Vulnerability Sync Tasks
-* 1.2.0 - Get device by IP and Get device by hostname: fix validation issue when IP or hostname not found | Add unit tests
-* 1.1.1 - Fix undefined org ID passed to actions when not required | Record outcome of connection tests
-* 1.1.0 - Add `report source` as optional input parameter to Upload Vulnerability Sync File action | Add report source to batch type
+  
+* 3.0.0 - `Action`: Fixed -  Get Vulnerability Sync Action Set | `Action`: Fixed - List Vulnerability Sync Action Sets |
+ `Action`: Updated - List Organization Users  
+* 2.0.0 - Fix Vulnerability Sync API Actions | `Action`: Added - Delete Vulnerability Sync Action Set | `Action`: Added 
+- Execute Vulnerability Sync Actions | `Action`: Added - List Vulnerability Sync Action Set Issues | `Action`: Added - 
+List Vulnerability Sync Action Set Solutions | `Action`: Added - List Vulnerability Sync Action Sets | `Action`: Added -
+ Get Vulnerability Sync Action Set | `Action`: Updated - Upload Vulnerability Sync File | `Action`: Updated - Get 
+Devices | `Action`: Deleted - Action on Vulnerability Sync Batch | `Action`: Deleted - Vulnerability Sync Task | 
+`Action`: Deleted - Get Vulnerability Sync Batch | `Action`: Deleted - List Vulnerability Sync Batches | `Action`: 
+Deleted - List Vulnerability Sync Tasks  
+* 1.2.0 - Get device by IP and Get device by hostname: fix validation issue when IP or hostname not found | Add unit 
+tests  
+* 1.1.1 - Fix undefined org ID passed to actions when not required | Record outcome of connection tests  
+* 1.1.0 - Add `report source` as optional input parameter to Upload Vulnerability Sync File action | Add report source 
+to batch type  
 * 1.0.0 - Initial plugin
 
 # Links
 
-* [Automox](https://www.automox.com/)
 
 ## References
-
-* [Automox Developer Portal](https://developer.automox.com/)
-* [Managing API Keys](https://support.automox.com/help/managing-keys)
-* [Event Types for Get Automox Events action](https://developer.automox.com/openapi/axconsole/operation/getEvents/#!in=query&path=eventName&t=request)
-
+  
+* [Automox](https://www.automox.com/)  
+* [Automox Developer Portal](https://developer.automox.com/developer-portal/)
```

Comment:
Please also add a link/links to the `Links` section 😄

Reply:
What is the correct way to do this in the plugin spec? I didn't manually add these.

Reply:
I just referenced this other plugin spec: https://github.com/rapid7/insightconnect-plugins/blob/master/plugins/abnormal_security/plugin.spec.yaml

I see that it has a link under the `Links` headers, but I don't see anywhere in the plugin spec that would lead to generating that output. When I manually add a link and run `insight-plugin refresh` the link I added is removed.

Reply:
You can add it manually to the `help.md` file